src/main/java/ejercicio/banco/dto/Bank.java

```diff
@@ -4,29 +4,28 @@
 
 public class Bank {
 
-    //Variables
-    private int bankId;
+    private int id;
     private String bankName;
     private String bankAddress;
```

Comment:
Here it happens the same: makes no sense to say `bankName` and `bankAddress`. Just `name` and `address` would be enough and more concise. 

Please, review all the dtos and apply this criteria.